h/static/scripts/test/streamer-test.js

```diff
@@ -75,42 +76,47 @@ describe('streamer', function () {
       websocketUrl: 'ws://example.com/ws',
     };
 
-    streamer = proxyquire('../streamer', {
+    Streamer = proxyquire('../streamer', {
       './websocket': FakeSocket,
     });
   });
 
   it('should not create a websocket connection if websocketUrl is not provided', function () {
     fakeSettings = {}
     createDefaultStreamer();
+    activeStreamer.connect();
     assert.isNull(fakeWebSocket);
   });
 
+  it('should not create a websocket connection if not explicitely connected', function () {
+    createDefaultStreamer();
+    assert.isNull(fakeWebSocket);
+    activeStreamer.connect();
+    assert.ok(fakeWebSocket);
+  })
+
   it('should send a client ID', function () {
     createDefaultStreamer();
+    activeStreamer.connect();
     assert.equal(fakeWebSocket.messages.length, 1);
     assert.equal(fakeWebSocket.messages[0].messageType, 'client_id');
-    assert.equal(fakeWebSocket.messages[0].value, streamer.clientId);
+    assert.equal(fakeWebSocket.messages[0].value, Streamer.clientId);
```

Comment:
This test must also be broken if it is passing since I assume that `Streamer.clientId` is undefined here.